tests/test_var.py

```diff
@@ -26,37 +26,39 @@ def test_List(self):
 
         idx = v.sample(rng=rng)
         assert idx == 2
-        
+
         idx = v.sample(size=2, rng=rng)
         assert all(x1 == x2 for x1, x2 in zip(idx, [0, 2]))
 
         x1 = mpy.List([0, 1, 2])
         x2 = mpy.List([0, 1, 2])
         assert x1 == x2
 
+        cat_ordinal = mpy.List(values=["low", "medium", "high"], name="cat_ordinal")
+        cat_nominal = mpy.List(values=["red", "blue", "green", "yellow"], name="cat_nominal")
```

Comment:
these 2 tests do not check much... what needs to be checked/ensured? where is `ordered`, use the `assert` statement

Reply:
This was a mistake. I shall add it. Apologies